spec/helpers/application_helper/buttons/vm_instance_template_scan_spec.rb

```diff
@@ -40,15 +43,15 @@
       before do
         roles = %w(smartproxy smartstate).collect { |role| FactoryGirl.create(:server_role, :name => role) }
         FactoryGirl.create(:miq_server, :zone => MiqServer.my_server.zone, :active_roles => roles)
-        button.calculate_properties
+        subject.calculate_properties
       end
 
       context 'when record has active proxy' do
-        it_behaves_like 'an enabled button'
+        include_examples 'ApplicationHelper::Button::Basic enabled'
       end
       context 'when record does not have active proxy' do
         let(:has_active_proxy?) { false }
-        it_behaves_like 'a disabled button', 'No active SmartProxies found to analyze this VM'
+        include_examples 'ApplicationHelper::Button::Basic disabled', 'No active SmartProxies found to analyze this VM'
```

Comment:
Okay and how should anyone else know this? (use a hash with the rightly chosen key)